configs/plugins/groups/rt_slam.yaml

```diff
@@ -0,0 +1,3 @@
+plugin_group:
+  - !include "plugins/offline_imu_cam.yaml"
+  - !include "plugins/pose_prediction.yaml"
```

Comment:
Refactoring for the plugin groups (like this one) seems necessary to make things simpler. The existing group yaml files with matching names to the old files (e.g. `plugins/groups/rt_slam.yaml` and `rt_slam_plugins.yaml`) are the result of factoring out the common plugins into new yaml files based on their usage across the action yaml files. However, the plugins left behind don't seem to match the purpose of their original plugin group's name.